src/component/Registration/Signup.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './Signup.css'
+import NavBar from '../../component/NavBar/NavBar'
+import Footer from '../../component/Footer/Footer'
+
+
+const Signup = () => {
+  return <>
+           <NavBar />
+            <section> 
+                <div className="container">
+                    <form action="" className="form">
+                        <h1 className="signuptitle">Register</h1>
+
+                        <div className="form-group ">
+                            <label for="" className="label">Name</label>
+                            <input type="text" className="input" placeholder="" />
+
+                        </div>
+
+                        <div className="form-group">
+                            <label for="" className="label">Username</label>
+                            <input type="text" class="input" placeholder="" />
+
+                        </div>
```

Comment:
I suggest the form-group should be extracted into component to avoid DRY principle

Reply:
done also